alertmanager/client/client_test.go

```diff
@@ -240,7 +240,7 @@ func newTestClient() (AlertmanagerClient, *mocks.FSClient, *[]byte) {
 	tenancy := &alert.TenancyConfig{
 		RestrictorLabel: "tenantID",
 	}
-	return NewClient("test/alertmanager.yml", "alertmanager-host:9093", tenancy, fsClient), fsClient, &outputFile
+	return NewClient("test/alertmanager.yml", "alertmanager-host:9093", tenancy, fsClient, false), fsClient, &outputFile
```

Comment:
It seems like this is getting pretty long, and seeing "false" passed in to a function is difficult from a readers perspective.  Should we pass a config object to this?